src/Utils/DEM.jl

```diff
@@ -0,0 +1,77 @@
+
+
+abstract type DEM end
+
+############# DEM Interface begin ######################################
+
+"""
+    getCoordinate(dem::T,index::Tuple{Integer,Integer}) where T <: DEM
+
+    Get the coordinate for a certain index in the DEM. 
+    (All the DEM's are stored as a matrix of heights)
+"""
+function getCoordinate(dem::T,index::Tuple{Integer,Integer}) where T <: DEM
+     ErrorException("Not implemented for the following DEM type, $T")
+end
+
+
+"""
+    getIndex(dem::T,coordinate::Tuple{Real,Real}) where T <: DEM
+
+    Get the index of the DEM corresponding to the coordinate. 
+    (All the DEM's are stored as a matrix of heights)
+"""
+function getIndex(dem::T,coordinate::Tuple{Real,Real}) where T <: DEM
+    ErrorException("Not implemented for the following DEM type, $T")
+end
+
```

Comment:
What exactly is the functionality of this function. Is it only added to give an error or is is because you want to implement the following DEM type later? It yes to the latter, it overkill to prepare for every type?

Reply:
Also just the inteface